test/functional/legacy/scroll_opt_spec.lua

```diff
@@ -780,6 +773,38 @@ describe('smoothscroll', function()
     ]])
   end)
 
+  -- oldtest: Test_normal_j_below_botline()
+  it([[disabled "j" does not skip lines when scrolling below botline and 'foldmethod' is not "manual"]], function()
```

Comment:
Why is there a "disabled" in this description

Reply:
Oh, I think it's better to move it out of this `describe()` block then.

Reply:
I moved it to a new file then